Modules/Registration/Metricsv4/wrapping/itkMetricv4Types.wrap

```diff
@@ -71,12 +71,12 @@ foreach(z ${pointsets})
 endforeach()
 endmacro()
 
-_add_pointset_types("EDPSTPSM" "itk::EuclideanDistancePointSetToPointSetMetricv4" ${_real_pointsets})
-_add_pointset_types("EBPSTPSM" "itk::ExpectationBasedPointSetToPointSetMetricv4" ${_real_pointsets})
-_add_pointset_types("JHCTPSTPSM" "itk::JensenHavrdaCharvatTsallisPointSetToPointSetMetricv4" ${_real_pointsets})
-_add_pointset_types("PSTPSM" "itk::PointSetToPointSetMetricv4" ${_real_pointsets})
-_add_pointset_types("PSTPSMWI" "itk::PointSetToPointSetMetricWithIndexv4" ${_real_pointsets})
-_add_pointset_types("LPSTPSM" "itk::LabeledPointSetToPointSetMetricv4" ${_int_pointsets})
+_add_pointset_types("EDPSTPSM" "itk::EuclideanDistancePointSetToPointSetMetricv4" "${_real_pointsets}")
```

Comment:
Man, the missing double-quotes in CMake have bit me before.  That would explain the behavior mentioned.

Reply:
I have definitely done this as well, no worries 😅 Neat to see the sub-list processing in your macro here.